src/module/dom-utils.js

```diff
@@ -0,0 +1,31 @@
+import { API_URL, tvWrapper } from './constant.js';
+import getData from './api-util.js';
+
+const displayAllData = (data) => {
+  data.forEach((element) => {
+    tvWrapper.innerHTML += `
+         <div class="card" id=${element.id}> 
+           
+           <img src=${element.image.medium} alt ="best" class="image"/>
+           <div class="show-info">
+            <p class="show-name"> ${element.name}</p>
+            <div class="like">
+            <i class="fa fa-heart heart" aria-hidden="true"></i>
+            <div class="likes-num">
+               <span> ${data.length}</span> likes
+            </div>
```

Comment:
Maybe after the likes-num div you can add a comments button to display the popup section, the rest looks nice